CombinatorialGames/Game/Ordinal.lean

```diff
@@ -76,11 +76,11 @@ theorem toPGame_moveLeft {o : Ordinal} (i) :
     o.toPGame.moveLeft (toLeftMovesToPGame i) = i.val.toPGame := by simp
 
 /-- `0.toPGame` has the same moves as `0`. -/
-noncomputable def zeroToPGameRelabelling : toPGame 0 ≡r 0 :=
-  Relabelling.isEmpty _
+lemma zero_toPGame : toPGame 0 ≡ 0 :=
```

Comment:
```suggestion
lemma toPGame_zero : toPGame 0 ≡ 0 :=
```

Reply:
I should note it's a bit unorthodox to name a theorem like this (it'd be more usual to name it `toPGame_zero_identical`) but since identity is "basically" equality then I don't mind this convention.